### DISCUSSION TITLE
Common Yatai-Service shared across multiple groups in an org

### DISCUSSION THREAD

**Discussion by jiyer2016** [bentoml/BentoML#1176] [Ideas] [answered]
I was reviewing - https://github.com/bentoml/BentoML/discussions/1128 - and am interested to learn more about this item:
`0.10.0 Add documentation on best practices for running a shared YataiService for team collaboration `

Would this provide the ability for multiple teams in an org to work off a single yatai-service ?
It may be OK for each team to view a list of models belonging to other teams
However - the ability to store / download models needs to be restricted to within the team

I know we don't have a concept of "user" and "groups" in BentoML yet - But do we have any such plans in the longer term ?

**Comment by parano** [comment, marked as answer]
Great question @jiyer2016 -  We are improving the overall model management features, for example, PR https://github.com/bentoml/BentoML/pull/1126 makes it a lot easier to create CI/CD workflow with other CI or deployment tools, and for Data Scientists to share and access their models stored in a central shared YataiService.

We currently do not have plans to add "users" and "groups" in open source BentoML, that along with authentication and authorization will be available in our [commercial product to come](HTTP://bentoml.ai) for enterprise customers. As you may know, we are a startup company building an open core business around this open-source project.  Our plan is to build enterprise features around the open-source core functionalities and charge enterprise customers for a premium.  And by building a sustainable business around the project this way, the BentoML core team will be able to continue to build and maintain the core model serving & deployment functionalities for the open-source community.
